### PR TITLE
nit: further research, task 3 fix

### PR DESCRIPTION
**Pull request by yurijmikhalevich**
In the example we have the following code:
```python
def conv(ni, nf, ks=3, act=True):
    layers = [nn.Conv2d(ni, nf, stride=2, kernel_size=ks, padding=ks//2)]
    layers.append(nn.BatchNorm2d(nf))
    if act: layers.append(nn.ReLU())
    return nn.Sequential(*layers)
```

We have batch norm added before activation, therefore the task should say "try moving batch norm after the activation…".

It looks like we either do this or update the example.

### CODE REVIEW

**Comment by yurijmikhalevich** [comment]
Judging by the text it's better to update an example.

```
Batch normalization (often just called batchnorm) works by taking an average of the mean and standard deviations of the activations of a layer and using those to normalize the activations. However, this can cause problems because the network might want some activations to be really high in order to make accurate predictions. So they also added two learnable parameters (meaning they will be updated in the SGD step), usually called gamma and beta. After normalizing the activations to get some new activation vector y, a batchnorm layer returns gamma*y + beta.
```
